Don't warn to book emergency or immediate repairs.

pudiva opened the pull request:
Emergency and Immediate repairs are not raised by agents.

pudiva committed:
Don't warn to book emergency or immediate repairs.

Emergency and Immediate repairs are not raised by agents.

pudiva committed:
Tell agents to check if appointment can be booked.

cdccollins commented on app/views/repair_requests/created.html.erb:
```diff
@@ -20,7 +20,12 @@
             <%= link_to "open DRS", drs_url, target: "_blank" %> to book an appointment!
           <% else %>
-            open UH to book an appointment!
+            open UH
```
I don't think this is what this story is asking.  If it's Immediate or Emergency priority then they should be opening DRS to check if the appointment has gone through, but not book it. They shouldn't be going to UH to book appointments.

pudiva replied:
Should be good now!

pudiva committed:
Fix duplicated "to book an appointment!" string.

Full message was:

> Please, open DRS to book an appointment! to book an appointment!

pudiva committed:
Change appointment warning when priority is high.
